src/Host/Client/Impl/Definitions/GuidList.cs

```diff
@@ -6,5 +6,7 @@
 namespace Microsoft.R.Host.Client {
     public static class GuidList {
         public static readonly Guid InteractiveWindowRSessionGuid = new Guid("77E2BCD9-BEED-47EF-B51E-2B892260ECA7");
+        public static readonly Guid PackageManagerRSessionGuid = new Guid("61C93E8D-D24D-4012-82F4-093086A4FB08");
```

Comment:
No, it actually should be the opposite - `InteractiveWindowRSessionGuid` should be removed from the `Microsoft.R.Host.Client`. And actually right now nothing prevents us from doing it